spec/features/workflow/publish_without_selection_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.feature "Attempting to publish without confirming a review status" do
```

Comment:
I would remove "Attempting to" here.  I'd also rename the file to be a bit clearer about what we're testing.  Something like `publish_without_selecting_review_status_spec.rb` perhaps?

Reply:
@emmabeynon we had a long think about this yesterday, and were trying to find a naming pattern that could also be used for the `choose_document_spec`, which does exactly the same kind of check. Taking some inspiration from @1pretz1's choice of locale, how about `publish_status_not_selected_spec.rb`, which could work as `create_document_format_not_selected_spec.rb`?

Reply:
Sounds good, perhapa `publish_review_status_not_selected_spec.rb` just to be really clear?  `create_document_format_not_selected_spec.rb` works for me 👍

Reply:
Fine with me :-).

Reply:
Yep i think Emma's last suggestion is a good file name to use, i'll make the changes! And i'll also open a separate pr for renaming the `choose_document_spec` to follow suite